kolibri/core/webpack/management/commands/devserver.py

```diff
@@ -36,13 +39,6 @@ def add_arguments(self, parser):
             default=False,
             help='Tells Django runserver to spawn a webpack watch subprocess.',
         )
-        parser.add_argument(
-            '--lint',
-            action='store_true',
-            dest='lint',
-            default=False,
-            help='Tells Django runserver to run the linting option on webpack subprocess.',
-        )
```

Comment:
Instead of just deleting this, maybe have it show a warning when `--lint` is passed?

e.g.

```
WARNING: The --lint argument is deprecated. Linting will always be run when the --webpack argument is used.
```

Instead of

```
kolibri --debug manage devserver --webpack --lint
WARNING:root:No C Extensions available for this platform.

INFO     Running Kolibri with the following settings: kolibri.deployment.default.settings.base
INFO:kolibri.core.apps:Running Kolibri with the following settings: kolibri.deployment.default.settings.base
usage: kolibri manage devserver [-h] [--version] [-v {0,1,2,3}]
                                [--settings SETTINGS]
                                [--pythonpath PYTHONPATH] [--traceback]
                                [--no-color] [--webpack] [--karma] [--ipv6]
                                [--nothreading] [--noreload] [--nostatic]
                                [--insecure]
                                [addrport]
kolibri manage devserver: error: unrecognized arguments: --lint
```